airflow/www/static/js/dags.js

```diff
@@ -204,33 +204,41 @@ function drawDagStatsForDag(dagId, states) {
     });
 
   g.append('svg:a')
-    .attr('href', (d) => `${dagRunUrl}?_flt_3_dag_id=${dagId}&_flt_3_state=${d.state}`)
+    .attr('href', (d) => {
+      const params = new URLSearchParams();
+      params.append('_flt_3_dag_id', dagId);
+      /* eslint no-unused-expressions: ["error", { "allowTernary": true }] */
+      d.state ? params.append('_flt_3_state', d.state) : params.append('_flt_8_state', '');
```

Comment:
```suggestion
      params.append('_flt_3_state', d.state || '');
```
I think this would be simpler, no? and then we don't need the eslint exception?

Reply:
Sorry, we can't do this.
See, #26584 
If d.state is null, use params `_flt_8_state` , mean `Is Null`, else use `_flt_3_state `, mean `Equal to`.

Of course, we can use `if ... else ...` statement block to  shake off eslint exception.
But I still think use ternary expression maybe more suitable in it.

Reply:
Oh yes, my bad. I didn't realse the 3 vs the 8.